aip/aog/3021.md

```diff
@@ -0,0 +1,181 @@
+---
+aip:
+  id: 3021
+  state: draft
+  created: 2019-10-04
+  updated: 2019-10-04
+  type: guidance
+js:
+  - /assets/js/graphviz/viz.js
+  - /assets/js/graphviz/lite.render.js
+  - /assets/js/aip/aip-graphviz.js
+permalink: /3021
+---
+
+# Type protos in Actions on Google APIs
+
+Many Actions on Google fulfillment APIs need to represent typed data (datetime,
+money, etc.). In some cases, there are pre-existing common components for
```

Comment:
Nitpick: "pre-existing" and "common" are somewhat duplicative.

Reply:
Fixed.